scripts/linters/codeowner_linter.py

```diff
@@ -211,7 +211,13 @@ def check_codeowner_file(self):
                 inside_blanket_codeowners_section = False
                 continue
             if stripped_line and stripped_line[0] != '#':
-                if '@' not in line:
+                if '#' in line:
+                    error_message = (
+                        '%s --> Please remove inline comment from line %s' % (
+                            CODEOWNER_FILEPATH, line_num + 1))
+                    self.error_messages.append(error_message)
+                    self.failed = True
+                elif '@' not in line:
```

Comment:
What if there are no code owner in a line as well as contains inline comment do we want to raise two error message? (I think it should be 'if' instead of 'elif'.)